app/views/qing_groups/_modal_body.html.erb

```diff
@@ -23,5 +23,9 @@
     <%= f.field(:display_logic,
           partial: "questionings/display_logic",
           locals: {form_item: @qing_group}) %>
+    
+    <%= f.field(:repeat_count,
+      label: t("qing_group.repeat_count"),
+      hint: t("activerecord.hints.qing_group.repeat_count")) if @qing_group.repeatable? %>
```

Comment:
this field should probably be hidden if you uncheck "Repeatable?" just like "Repeat Item Name" gets hidden. idk where that logic is exactly, likely javascript?

Reply:
oooh could we also make this a dropdown, filtered by only **integer or counter** questions in the form? we're gonna have no end of support requests if people try to enter invalid XML here

Reply:
agreed re: hidden. 

yep, that would be ideal re: having a drop down with filtered questions. I won't have time to do this before I leave, so feel free to take it forward!